src/sdp/SDP.ts

```diff
@@ -927,8 +927,9 @@ export default class SDP {
         // `x-google-start-bitrate` is an unofficial flag that has existed in libwebrtc since its release and is unlikely
         // to be removed without notification.
         //
-        // libwebrtc reference: https://webrtc.googlesource.com/src/+/b6ef1a736ee94d97cc28f3bd59b826c716a3278f/media/base/media_constants.cc#97
-        const newLine = line + `;x-google-start-bitrate=${bitrateKbps * 1000}`;
```

Comment:
oopsie daisy

Reply:
Wow, wonder how this would have worked so far with * 1000 already added.

Reply:
It just uses the max bitrate configured, so nothing terrible.